src/collection/views/EditCollection.tsx

```diff
@@ -83,6 +83,11 @@ const EditCollection: FunctionComponent<EditCollectionProps> = props => {
 		},
 	];
 
+	const onUnload = (event: any) => {
```

Comment:
Ook als je van route verandert en er unsaved changes zijn zou je niet verder mogen gaan.
Weet niet of dat in de scope van dit ticket zit maar mag zeker ook niet vergeten worden.

Reply:
Ik maak hier een apart ticket voor aan.